config.py

```diff
@@ -325,6 +325,14 @@ def clean_dd_url(url):
     return url[:-1] if url.endswith('/') else url
 
 
+def remove_empty(string_array):
+    emptyless = []
+    for el in string_array:
+        if el:
+            emptyless.append(el)
+    return emptyless
```

Comment:
we could shorten the function to `return filter(lambda x: x, string_array)`, but it might be less readable, up to you